spec/provider_spec.rb

```diff
@@ -363,11 +357,5 @@
         and_return(valid_response(:json))
       @viddler.get(example_url(:viddler))
     end
-
-    it "handles the :timeout option" do
```

Comment:
Non-blocking: Instead of removing this expectation, my preference is to mark it as pending by adding `:pending=>true` so that we don't lose track of this being a case that would be good to test eventually.

An example in another test file: https://github.com/ruby-oembed/ruby-oembed/blob/cd3f3531e3b0d1d0dec833af6f2b5142fc35be0f/spec/provider_spec.rb#L87